client/Rating&Reviews/Body.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
I had to install a new dependency here to satisfy the linter. It wants us to validate prop types specifically to avoid unexpected input. There are examples of how this is used at the bottom of most files.

Reply:
I ran into this also.